simpleSequential/test_floorDisivion.py

```diff
@@ -0,0 +1,18 @@
+import unittest
+from floorDivision import floorDivision
+
+class TestFloorDivision(unittest.TestCase):
+    def test_twoPositiveNumberInput(self):
+        self.assertEqual(floorDivision(10, 3), 3)
+
+    def test_twoNegativeNumberInput(self):
+        self.assertEqual(floorDivision(-7, -2), 3)
+
+    def test_onePostiveAndOneNegativeInput(self):
+        self.assertEqual(floorDivision(10, -2), -5)
+
+    def test_divisibleNumberInput(self):
```

Comment:
I like the idea of this test. But I still debate about the test description. It makes me feel something different. Can you ask chatGPT to have a better English description?

Reply:
Please review the old comments.

Reply:
Can you review this commit? Like I said “divisible number input” don’t seem right.